### PR TITLE
13.0 mig partner contact personal information page

### PR DESCRIPTION
**Pull request by jeroenev**

### CODE REVIEW

**Review by StephanRozendaal — APPROVED** [review]
LGTM

**Review by Daemo00 — APPROVED** [review]
Thanks! Looks good to me (code review)

**Comment by Daemo00** [comment]
@jeroen7s could you have a look at Travis? It is also suggesting the change to be made in https://travis-ci.org/OCA/partner-contact/jobs/597517397#L310-L323

File: partner_contact_personal_information_page/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline partner_contact_personal_information_page/__init__.py]: Make all licence lines https:

File: partner_contact_personal_information_page/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "summary": "Add a page to contacts form to put personal information",
+    "version": "13.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "Nicolas JEUDY,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto_install": False,
+    "depends": ["base"],
```
Comment (NL66278) [inline partner_contact_personal_information_page/__manifest__.py:14]: Indent from the beginning, for easier extention. (for depends and data keys).

File: partner_contact_personal_information_page/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This module extends the contact management functionality.
+
+Its only purpose is to serve as a base for other modules that add personal
+information fields to **contacts that are persons**.
+
```
Comment (NL66278) [inline partner_contact_personal_information_page/readme/DESCRIPTION.rst]: Better remove the next three lines. Module has proven continuing usefullness.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: LGTM but few changes should bring the module completely up to current OCA guidelines.

File: partner_contact_personal_information_page/__manifest__.py (hunk 2 above)
Comment (jeroenev) [reply partner_contact_personal_information_page/__manifest__.py:14]: Not possible.
Black will reformat this and pre-commit + Travis will fail.

Comment (jeroenev) [reply partner_contact_personal_information_page/__manifest__.py:14]: See: https://travis-ci.org/OCA/partner-contact/jobs/597517397#L310-L323

Comment (NL66278) [reply partner_contact_personal_information_page/__manifest__.py:14]: OK, accepted

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**